src/github/index.ts

```diff
@@ -11,9 +11,12 @@ import statsd from "../config/statsd";
 import { getLogger } from "../config/logger";
 import { metricWebhooks } from "../config/metric-names";
 import { Application } from "probot";
+import { logExpressErrorResponse } from "../util/log-express-error-response";
 
-export default (robot:Application) => {
+export default (robot: Application) => {
 	const logger = getLogger("github.webhooks");
+	// Catch non successful responses
+	robot.router.use(logExpressErrorResponse(logger));
```

Comment:
I wonder if we can use context.log, which has context id, added to the context on the line 25?

Reply:
line 25 is for the webhook handler, this is for the problem express router and we're logging the request which would have the context id I believe.